search-sentiments-map/test/country-trends-test.js

```diff
@@ -0,0 +1,189 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+const mocha = require('mocha');
+const chai = require('chai');
+const assert = chai.assert;
+const sinon = require('sinon');
+const countryTrends = require('./../routes/country-trends').countryTrends;
+const {Datastore} = require('@google-cloud/datastore');
+const datastore = new Datastore();
+const fetch = require('node-fetch'); // Used to access custom search.
+
+describe('Country Trends', function() {
+  describe('retrieveCountryTrends', function() {
+    const RETRIEVE_RESULTS_TIME_MS = 70 * 60000;
+    const CURRENT_DATA_TIME_RANGE_12_HOURS_MS = 12 * 60 * 60000;
+    const THREE_HALVES_TIME_RANGE = 1.5;
+    const THREE_TIME_RANGE = 3;
+    let currentTime;
+    let mockData;
+    let trendsByCountryThreeHalvesTimeRange;
+    let trendsByCountryThreeTimeRange;
+    let mockThreeHalvesTimeRangeUsResult;
+    let mockThreeTimeRangeUsResult;
+    let mockThreeHalvesTimeRangeUaResult;
+    let mockThreeTimeRangeUaResult;
+    let mockEmptyResult;
```

Comment:
The only consts/variables that should be declared out here are the ones that are called in the beforeEach block AND an "it" block. If it is only called within one block, it can be declared there instead.